addons/website_slides/models/slide_slide.py

```diff
@@ -877,16 +897,48 @@ def _action_set_completed(self, target_partner):
             'vote': 0,
             'completed': True} for new_slide in new_slides])
 
+        self.env['slide.channel'].invalidate_cache(['completed', 'completion'], ids=self.channel_id.ids)
```

Comment:
Is this necessary?
Writing on "completed" here above should trigger "_recompute_completion" normally.